package.json

```diff
@@ -100,5 +97,25 @@
     "ts-loader": "^9.5.1",
     "typescript": "^5.0.4",
     "webpack": "^5.90.3"
+  },
+  "packageManager": "pnpm@8.15.6",
+  "engines": {
+    "node": "^20.0.0",
+    "pnpm": "^8.15.6"
+  },
+  "volta": {
+    "node": "20.12.2",
+    "pnpm": "8.15.6"
+  },
+  "pnpm": {
+    "overrides": {
+      "@types/react": "17.0.2",
+      "@types/react-dom": "17.0.2"
+    },
```

Comment:
In a separate PR, we should look into upgrading `react` and `react-dom` to latest, I don't see a reason why it would be locked to v17.